### PR TITLE
Update profile & landing page

### PR DESCRIPTION
**Pull request by amiraabouhadid**
### Changes
- update profile page to this [design](https://www.figma.com/proto/PzOjPW17TZ9tO13DGNqdH7/profile?type=design&node-id=1-2&t=FRAGQsh5qzXAlyVS-0&scaling=min-zoom&page-id=0%3A1)
- removed popup when user is logged in
- update landing page to this [design](https://www.figma.com/proto/KRuoLPftqoCykuHkbw6XmQ/Untitled?type=design&node-id=1-60&t=PDmRvccfShhr6B84-0&scaling=min-zoom&page-id=0%3A1)

### Related Issues

#1862 #1878 

### Screenshots
landing page when user is logged in:
image
profile page:
image
when user clicks on balance card when logged in this dialog appears
image

### CODE REVIEW

**Comment by AlaaElattar** [comment]
`Explore Grid Capacity` and `Explore Grid Nodes` aren't clickable when user is logged out.

**Comment by AhmedHanafy725** [comment]
the original page before logging in should still be the same.
Screenshot from 2024-01-08 17-05-06

**Comment by AhmedHanafy725** [comment]
when the screen is small, the QR code goes down 

Screenshot from 2024-01-08 17-14-21

**Comment by amiraabouhadid** [comment]
> when the screen is small, the QR code goes down
> 
> Screenshot from 2024-01-08 17-14-21

https://www.loom.com/share/9d26ba798aaa4a2faa2961adc0925c15?sid=777db492-a9ac-4483-a47e-580ba18bf5b5

**Comment by ehab-hassan** [comment]
@amiraabouhadid Thanks for the Fast work just a small comments 

modifications

- [ ] Font size same size as My Profile 1.25rem or 20px
- [ ] Use the same style border we used for all App even links at right 
- [ ] Use smae hove we use it at the Application page (with transparent green color)
- [ ] img width 100
- [ ] font size for Values 1.25rem or H6
- [ ] font size for Value titles 1rem or text-subtitle-1

File: packages/playground/src/App.vue [hunk 1]
```diff
@@ -214,6 +215,15 @@ const theme = useTheme();
 const navbarConfig = ref();
 
 const hasGrid = computed(() => !!gridStore.grid);
+watch(profileManager, () => {
+  {
+    if (profileManager.profile) {
+      openProfile.value = false;
+      $router.push("/overview");
```
Comment (AhmedHanafy725) [inline packages/playground/src/App.vue:222]: let's not push this page after login. it should use the current route and if the user logged in and was on the root `/`, show this landing page.

File: packages/playground/src/components/logged_in_landing.vue [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        text: "Deploy Full or Micro Virtual Machines from available images or custom ones on the Threefold Grid.",
+        path: "vms",
+      },
+      {
+        title: "Deploy Orchestrators",
+        icon: "mdi-group",
+        text: "Deploy Kubernetes clusters out of the box or Caprover app/database deployment & web server manager on the Threefold Grid.",
+        path: "orchestrators",
+      },
+      {
```
Comment (AhmedHanafy725) [inline packages/playground/src/components/logged_in_landing.vue:119]: Add a new card for deploying applications

**Comment by AhmedHanafy725** [comment]
clicking on balance now shows nothing, I suggest showing the QR code for the user to fund his wallet

**Review by ehab-hassan — CHANGES_REQUESTED** [review]
image

Make the button at right green background and the left outline
also make margin my-4 for button not mt-8 
the switch color button needed also for the old home page when user not login

image


and here need mb-2 for values number

**Comment by AlaaElattar** [comment]
Please fix build

**Review by ehab-hassan — CHANGES_REQUESTED** [review]
image
The close button not working when I need to go through without login

File: packages/playground/src/App.vue [hunk 3]
```diff
@@ -215,26 +205,23 @@ const theme = useTheme();
...
 watch(
   () => $route.meta,
   meta => {
     (document.title = "Threefold Dashboard" + (meta && "title" in meta ? ` | ${meta.title}` : ``)),
       (navbarConfig.value = meta.navbarConfig);
   },
 );
+function navigateToHome() {
+  return $router.push("/");
+}
```
Comment (MohamedElmdary) [inline packages/playground/src/App.vue:218]: Please revemoe `navigateToHome` if it's not used anywhere?

**Comment by MohamedElmdary** [comment]
image

is that bottom border required?

**Comment by MohamedElmdary** [comment]
image

push cards to the end of column and make spaces equal from right/left/bottom

File: packages/playground/src/components/logged_in_landing.vue [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+          <div
+            :style="{
+              backgroundImage: 'url(' + baseUrl + 'images/live-and-operational-background.png)',
+
+              backgroundSize: 'cover',
+              backgroundPosition: 'center center',
+              minHeight: '350px',
+            }"
+          />
+          <div class="d-flex flex-wrap justify-center" style="statcards">
```
Comment (MohamedElmdary) [inline packages/playground/src/components/logged_in_landing.vue:18]: I think u mean `class` instead of `style`?

**Comment by ehab-hassan** [comment]
> image
> 
> * [ ]  Remove that border
> * [ ]  Make numbers `bolder`
> * [ ]  Update color of cards' title to be a little gray (ask @ehab-hassan for color he used)

@MohamedElmdary please leave it as is the numbers are already bold no need to add line between the value and text
image

File: packages/playground/src/App.vue (hunk 3 above)
Comment (amiraabouhadid) [reply packages/playground/src/App.vue:218]: it is used on line 119

**Comment by amiraabouhadid** [comment]
> image
> 
> * [ ]  Remove that border
> * [ ]  Make numbers `bolder`
> * [ ]  Update color of cards' title to be a little gray (ask @ehab-hassan for color he used)

I am not sure why you have a bottom border in your view, none was added and none is shown to me
image
 
image

**Review by ehab-hassan — COMMENTED** [review]
for this
photo_2024-01-10_12-17-44
I discussed this with @AhmedHanafy725  and we agree that using this for funding is better and not show it after login automatically just when the user clicks on the balance
image

**Review by ehab-hassan — APPROVED** [review]

**Comment by ehab-hassan** [comment]
> > image
> > 
> > * [ ]   Remove that border
> > * [ ]   Make numbers `bolder`
> > * [ ]   Update color of cards' title to be a little gray (ask @ehab-hassan for color he used)
> 
> I am not sure why you have a bottom border in your view, none was added and none is shown to me image @ehab-hassan can confirm UI fits his requirements

all good just move the map with values down a bit .

**Review by ehab-hassan — APPROVED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**